homework/hackyourtemperature/app.js

```diff
@@ -0,0 +1,11 @@
+import express from 'express';
+import router from './router.js';
+const app = express();
+
+//Middleware
+app.set('view engine', 'handlebars');
+app.use(express.json());
+app.use(express.urlencoded({ extended: false }));
+app.use('/', router);
```

Comment:
Generally we use the sub-route title here instead of just a `/` like: 

```suggestion
app.use('/weather', router);
```

It makes it easier with multiple routers 😄